staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -83,6 +84,9 @@ type GenericAPIServer struct {
 	// minRequestTimeout is how short the request timeout can be.  This is used to build the RESTHandler
 	minRequestTimeout time.Duration
 
+	// shutdownTimeout is the timeout used for server shutdown.
```

Comment:
What does this mean? I.e., this is the maximum duration between X and Y, what is X and what is Y?

Reply:
This is the Server.Shutdown Context timeout.

Reply:
@hzxuzhonghu can you add a Godoc sentence explaining this high-level for people not familiar with the code deep in the apiserver?

Reply:
Please answer questions like this by adding comments, not by responding on github :)

Reply:
OK, shutdownTimeout is the maximum time  between all Shutdown and Shutdown return.   It specifies the wait time before  server.Shutdown(),  to prevent indefinite blocking by server.Shutdown().

Reply:
@sttts Then how to describe `shutdownTimeout` more detailed? 
>can you add a Godoc sentence explaining this high-level for people not familiar with the code deep in the apiserver?

Reply:
@sttts I update `shutdownTimeout ` to `ShutdownTimeout`, and add some description.